src/librustc_middle/ty/context.rs

```diff
@@ -1120,7 +1120,18 @@ impl<'tcx> TyCtxt<'tcx> {
             consts: common_consts,
             extern_crate_map: resolutions.extern_crate_map,
             trait_map,
-            export_map: resolutions.export_map,
+            export_map: resolutions
+                .export_map
+                .into_iter()
+                .map(|(k, v)| {
+                    (
+                        k,
+                        v.into_iter()
+                            .map(|e| e.map_id(|id| definitions.as_local_hir_id(id)))
```

Comment:
Side note: `as_local_hir_id` doesn't make much sense, all `HirId`s are local.
`fn as_local_hir_id` can be renamed to simply `fn as_hir_id`.

Reply:
(That's certainly for a separate PR.)

Reply:
I'll note that for a future refactoring. Its not the only method in the hir map that has a confusing name.